api/src/pcapi/routes/backoffice_v3/auth.py

```diff
@@ -70,35 +68,26 @@ def authorize() -> utils.BackofficeResponse:
         )
         return werkzeug.exceptions.Forbidden()
 
-    if settings.IS_TESTING or settings.IS_DEV or settings.IS_INTEGRATION:
-        if not user:
-            user = create_local_admin_user(
-                email=google_user["email"],
-                first_name=google_user["name"],
-                last_name=google_user["family_name"],
-                given_name=google_user["given_name"],
-            )
+    if not user:
+        user = create_local_admin_user(
```

Comment:
Selon ma compréhension du ticket, en staging/prod on ne doit créer le compte que s'il est dans au moins un groupe Google. Le SSO peut être valide sans être dans aucun des groupes de rôles.
Donc il me paraît plus cohérent avec la demande de vérifier les rôles reçus de GW avant de créer effectivement le compte admin.

Donc la partie « user not found » doit encore exister (toujours selon ma compréhension), pour le cas où un compte SSO pass Culture s'authentifie, mais sans avoir de compte dans la base de données ni être dans un groupe GW ; on ne veut pas forcément donner à cet utilisateur le rôle `ADMIN` qui donne un accès étendu à FA et PC Pro.